src/workerd/api/actor-alarms-delete-test.js

```diff
@@ -23,7 +23,9 @@ export class DurableObjectExample {
     let timeoutMs = (timeMs - Date.now().valueOf()) + (timeoutOverrideMs ?? 4000);
     try {
       await Promise.race([prom, timeout(timeoutMs)]);
-      if (Date.now() < scheduledTime.valueOf()) {
+      // We are giving a 10ms margin for the current time because scheduled alarm can run in tests a
```

Comment:
Delay appears to be 10ms, but comments also mention 2ms

Reply:
My bad, increased the value to avoid problem. Will correct that :)